src/assets/scripts/airport/Fix/FixModel.js

```diff
@@ -0,0 +1,114 @@
+import _cloneDeep from 'lodash/cloneDeep';
+import _uniqueId from 'lodash/uniqueId';
+import PositionModel from '../../base/PositionModel';
+
+/**
+ * Defines a navigational `FixModel`
+ *
+ * A `FixModel` can be used as part of a `StandardRoute` or as a naviagtional aid.
+ * Not all `FixModel`s are a part of a `StandardRoute`.
+ *
+ * @class FixModel
+ */
+export default class FixModel {
+    /**
+     * @for FixModel
+     * @constructor
+     * @param fixName {string}
+     * @param fixCoordinate {array}
+     * @param airportPosition {PositionModel}
+     */
+    constructor(fixName, fixCoordinate, airportPosition) {
+        if (!fixName || !fixCoordinate || !airportPosition) {
+            return;
+        }
+
+        /**
+         * Unigue string id that can be used to differentiate this model instance from another.
```

Comment:
Sorry if it's becoming annoying, but I'm curious again here: what is the need to differentiate between whether or not a given instance of a `FixModel` is a particular instance as oppposed to another? I assume there must be situations where that is helpful, but I'm not sure what an example would be.